certification/internal/policy/operator/scorecard_check.go

```diff
@@ -42,7 +45,7 @@ func (p *scorecardCheck) getDataToValidate(bundleImage string, selector []string
 		Namespace:      namespace,
 		ServiceAccount: serviceAccount,
 		Verbose:        true,
-		WaitTime:       "240s",
+		WaitTime:       fmt.Sprintf("%s%s", waitTime, "s"),
```

Comment:
Small nit, feel free to ignore, but... The second arg? Is that necessary?

Reply:
@bcrochet You're thinking`fmt.Sprintf("%ss", waitTime)` right? That's probably clearer/cleaner.

Reply:
Updated this for clarity.